autoload/importjs.vim

```diff
@@ -170,14 +170,10 @@ function importjs#ReplaceBuffer(content)
   " Save cursor position so that we can restore it later
   let cursorPos = getpos(".")
   let originalLineCount = line("$")
-  " Delete all lines from the buffer
-  execute "%d"
-  " Write the resulting content into the buffer
-  let @a = a:content
-  normal! G
-  execute "put a"
-  " Remove lingering line at the top:
-  execut ":1d"
+  "Overwrite the current file
+  execute 'call writefile(split(a:content, "\n"), expand("%"))'
+  "Reopen the file
+  execute 'silent! e!'
```

Comment:
This change will force the buffer to be written to file when we run import-js. The previous code here would simply update the buffer, which to me is a much better experience.

Reply:
Why is it a better experience? In what case would you want to fix the imports, but not save the changes. I find the lag when adding imports and the noticeable movement of the buffer annoying. I always save after fixing the imports in a file.

Reply:
>  In what case would you want to fix the imports, but not save the changes?

- when you're working on a new file, not previously stored (and you don't know yet it you want to save it). 
- when you want to batch multiple commands before you save the file. I often run `prettier` as well. 

The movement of the buffer is a separate issue, but I agree it's annoying.

Reply:
1) Why would you add code to a buffer, add the imports and not save after that? Why would you even add the code initially and not save? What is the big deal of creating a file - you can always remove it afterwards.
2) That is why you can modify the mapping that you use to fix the imports, the same way I have done, since I use folds.
Example: `nnoremap <silent> <leader>if :ImportJSFix<CR>:ALEFix<CR>:update<CR>:silent! normal! zO<CR>`
3) I don't even know how to respond to that. There is less than second before the file is saved. It is not possible for you to make any meaningful change in that time.
I bet you haven't even tried this change.

Btw, I use ALE and expression folding which occurs on save.
The fix imports command is insanely fast.

Reply:
You are making a lot of assumptions here about people's workflows that I'm not comfortable with. Can any of this be accomplished in your own vim configuration? If not, perhaps we could expose hooks or functions or something that would allow you to.

Reply:
I am tired of this. I don't see any "assumptions" that I have made about people's workflows. Either something makes sense or it doesn't. All three points I made are valid.
Either merge this and my other PR https://github.com/Galooshi/import-js/pull/506 or don't.
Contributing for those projects is annoying, so I am done.

Reply:
Sounds good, I'll close this PR then.